src/components/organisms/AddForm/AddForm.jsx

```diff
@@ -0,0 +1,71 @@
+import { Formik, Form, Field } from 'formik';
+import * as Yup from 'yup';
+import PropTypes from 'prop-types';
+import { Icon } from '../..';
+
+import './Form.scss';
+
+export const AddForm = ({ placeholder, type }) => {
```

Comment:
"AddForm" seems like not accurate name, just Form would be fine

Reply:
I tried this and got error something like "form is declared"

Reply:
Yeah, right, in scope, import etc is't declared,
You can just change the component name to Form, and import {Form as FormikForm} from 'formik'
The eslint won't yell at you anymore :P